common/shlibs

```diff
@@ -4238,3 +4238,5 @@ libdate-tz.so.3 chrono-date-3.0.1_1
 libayatana-ido3-0.4.so.0 ayatana-ido-0.9.2_1
 libayatana-indicator3.so.7 libayatana-indicator-0.9.3_1
 libayatana-appindicator3.so.1 libayatana-appindicator-0.5.91_1
+libcamera.so.0.0.4 rpi-libcamera-230303_1
+libcamera-base.so.0.0.4 rpi-libcamera-230303_1
```

Comment:
These are the same sonames as what the libcamera package has, so this won't work correctly once #41263 is merged.

> PLEASE NOTE: when multiple packages provide the same SONAME, the first one (order top->bottom) is preferred over the next ones.

Reply:
OK, so what's the solution of this? The user should be able to choose if they want libcamera, or rpi-libcamera (on Raspberry Pi). I understand that I should add the `conflicts` variable specifying libcamera package but how can we avoid the conflict in `common/shlibs`?